pkg/roachprod/roachprod.go

```diff
@@ -1074,6 +1074,28 @@ func AdminURL(
 	return urlGenerator(ctx, c, l, c.TargetNodes(), uConfig)
 }
 
+// SQLPorts finds the SQL ports for a cluster.
```

Comment:
Super Nit: This only supports the host/storage cluster at this point, and might be beneficial if it supports passing virtual cluster info in the future. Not required for this PR though.

Reply:
It's now passing the `tenant` name and the `sqlinstance` down to `DiscoverService`. made the change for `AdminUIPorts` too.

Nothing right now is actually passing a non `""` or `0` value for those, but I played around with it in the `multitenant/distsql` tests and it seemed like it worked to me. It was returning different port values depending on if I passed `tenantName` or not.